apps/commudle-admin/src/app/feature-modules/public-community/components/membership-toggle/membership-toggle.component.html

```diff
@@ -1,5 +1,6 @@
+<!-- (click)="openDialog(addMemberDialog)" -->
```

Comment:
why are we comment this?

Reply:
I have not deleted any previous function, just commented it out, once the testing is done, i will remove the function through which the component was working earlier